content/en/apps/_index.md

```diff
@@ -4,20 +4,13 @@ linkTitle: "CHT Applications"
 identifier: "apps"
 weight: 2
 description: >
-  Overview and reference for building digital health apps
+  Overview and reference for building CHT applications
 ---
 
-{{% pageinfo %}}
-This section provides an overview and reference for building digital health applications with CHT Core.
-{{% /pageinfo %}}
+To create a digital health tool using the Community Health Toolkit: build a CHT Application or re-use an existing CHT Application. The [CHT Core Framework]({{ ref="core" }}) provides a foundation on which many digital health tools can be conceptualised and built. A CHT Application takes that foundation and builds on it to create a concrete digital health tool tailored to the health workflows in a targeted community. 
 
-From a technical perspective, developing a custom app begins with writing XForms, JSON, and JavaScript code that configures the Core Framework’s features to meet your organization’s needs. 
+Since all CHT Applications share the same foundation, they also share capabilities and attributes. For example, all CHT Applications share a similar user-interface and share the same look-and-feel. All CHT Applications can be built to scale, can run across a variety of hardware, support offline-first experiences, and support most languages. These baseline capabilities are universal across all CHT Applications. Learn more about this in [Why the CHT?]({{ ref="why-the-cht" }})
 
-The CHT Core Framework allows you to define each element in your app in a modular way, and then specify when and how it should appear for different types of users, without having to modify the underlying framework. Collectively, this customization is referred to as Configuration Code.
+Although CHT Applications share this common foundation, they can be built to tackle the diverse health challenges of communities around the globe. Some examples of health interventions which have been digitized by CHT applications include antenatal care, postnatal care, integrated community case management, family planning, malnutrition, immunization, and epidemic response. 
```

Comment:
So my thinking was:

Paragraph 1-- very roughly, what is a CHT Application? It is a digital health tool following a specific pattern
Paragraph 2 -- what is the framework anyway? What is the relationship between the framework and the app? how are cht apps similar?
Paragraph 3 (this paragraph) --  what sort of things can apps do? (how is each one unique and highly contextualised to a specific problem)

If we are going to jump into the idea of "profiles for families" I think we need to set the stage quite a bit before that. Do readers know that these apps are centered around filling forms, completing workflows, and saving that data? I think contact profiles are too deep to explain right away... So, I'd skip your first sentence.

But I love the people first and some other edits. What do you think about touching on cases where our users are not CHWs or CHW Supervisors (like MoH-Covid-KE or Goma) for example? We focus on CHWs, but the tool isn't exclusive to them. I'm not sure what is the right way to message that, if at all.

Reply:
I really like the changes overall, the language around CHT applications and CHT Core as a foundation seems clearer. On this part:

>  CHT apps have been used by CHWs supporting their communities with a variety of health aspects...

I agree with @abbyad's feedback, yet it might benefit from further iteration. Perhaps like:
> Custom CHT applications have been built to support a range of health interventions, including antenatal care, postnatal care, family planning, integrated community case management of childhood illness, malnutrition, immunization, and epidemic response. 

Happy to see this continue to improve!

Reply:
> If we are going to jump into the idea of "profiles for families" I think we need to set the stage quite a bit before that. Do readers know that these apps are centered around filling forms, completing workflows, and saving that data? I think contact profiles are too deep to explain right away... So, I'd skip your first sentence and leave that to the top-level paragraphs on the "Features" page.

In the technical docs it makes sense to me to focus first on the technical components and how they come together. The features page and some top level pages seem like good places to focus first on describing health records or task management when an audience is familiar with these feature sets in other digital health software, but not necessarily as concerned with the the technical components of CHT apps. 

> What do you think about touching on cases where our users are not CHWs or CHW Supervisors (like MoH-Covid-KE or Goma) for example? We focus on CHWs, but the tool isn't exclusive to them. I'm not sure what is the right way to message that, if at all and that is why I ended up on the more generic "health tool"

I'm all for describing for the more generic user in our docs, for example we also have CHT applications that are used by a mix of nurses and patients and don't involve any CHWs, and this is still very much under the umbrella of a "community health system". I think we can refer to _health workers_ using _CHT apps/digital health apps/digital tools_, but _health tool_ might be a little too generic.